### PR TITLE
Add ocr pill with query for exact text search

### PR DESCRIPTION
**Pull request by AlexOVeritone**
1. Add Recognized Text pill component that does the exact text search.
2. Correct css to place pills in row - currently stacks in column
3. Change transcript modal text to be the same as in discovery-app

### CODE REVIEW

File: packages/veritone-react-common/src/components/SearchBar/story.js [hunk 1]
```diff
@@ -20,15 +26,29 @@ const transcript = {
   showPill: true
 };
 
+const recognizedText = {
```
Comment (SteveShaffer) [inline packages/veritone-react-common/src/components/SearchBar/story.js]: @atungw Why are we doing another key for each engine category?  Why not just add these attributes to `engineCategoryMapping` below and pass the engine category IDs as the `enabledEngineCategories`?

File: packages/veritone-react-common/src/components/TranscriptSearchModal/index.js [hunk 2]
```diff
@@ -67,7 +67,7 @@ export default class TranscriptSearchModal extends React.Component {
 export const TranscriptSearchForm = ( { defaultValue, cancel, onSubmit, onChange, onKeyPress, inputValue } ) => {
   return (
   <div>
-    <DialogTitle>Search Transcript</DialogTitle>
+    <DialogTitle>Search by Keyword</DialogTitle>
```
Comment (SteveShaffer) [inline packages/veritone-react-common/src/components/TranscriptSearchModal/index.js]: I'm not convinced we shouldn't take this opportunity to "generalize" this stuff.

Let's leave this for now.  But at the end of this I'll probably take a pass through here and tweak the copy to be more in line with our generalized, non-media-specific data model.

Comment (atungw) [reply packages/veritone-react-common/src/components/TranscriptSearchModal/index.js]: FYI, just recapping our last conversation but most of that information is defined here and will be passed into the SearchBar to configure it.

See the endpoint: https://enterprise.stage.veritone.com/api/engine/category?time=1517268957867

File: packages/veritone-react-common/src/components/SearchBar/story.js (hunk 1 above)
Comment (atungw) [reply packages/veritone-react-common/src/components/SearchBar/story.js]: This stuff is hardcoded for convenience, but it'll mostly be configured from here. https://enterprise.stage.veritone.com/api/engine/category?time=1517268957867

**Pull request merged**